ui/lib/core/addon/components/toggle-button.hbs

```diff
@@ -3,6 +3,7 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
+{{! DEPRECATED use <Hds::Reveal /> instead https://helios.hashicorp.design/components/reveal }}
```

Comment:
replacing this broke a lot of stuff, so added this comment instead, hopefully we can eventually replace all instances with `<Hds::Reveal>` and remove this component all together, but it's currently used in lots of forms and is out of scope for this PR